packages/iov-ethereum/src/ethereumconnection.spec.ts

```diff
@@ -246,10 +248,9 @@ describe("EthereumConnection", () => {
       const result = resultSearch[0];
       expect(result.transactionId).toEqual(resultPost.transactionId);
       const transaction = result.transaction;
-      if (transaction.kind !== TransactionKind.Send) {
+      if (!isSendTransaction(transaction)) {
```

Comment:
This custom `isSendTransaction` for every blockchain could be avoided if we just use the same domain for all `SendTransactions`. Or even just remove the whole domain idea, and hope we choose globally unique `kind`s

Reply:
`isSendTransaction` is from BCP, not implemented per blockchain. It gives you a `SendTransaction`, which is an interface that describes a shape of a transaction with no domain included. `SendTransaction` just required the `kind` to equal "send"